<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Logined</class>
 <widget class="QDialog" name="Logined">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>338</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>441</width>
     <height>341</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Profile</string>
    </attribute>
    <widget class="QTabWidget" name="tabWidget_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>441</width>
       <height>321</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Information</string>
      </attribute>
      <widget class="QGroupBox" name="groupBox">
       <property name="geometry">
        <rect>
         <x>5</x>
         <y>4</y>
         <width>401</width>
         <height>171</height>
        </rect>
       </property>
       <property name="title">
        <string>Informations of Users</string>
       </property>
       <widget class="QLineEdit" name="lineNickname">
        <property name="geometry">
         <rect>
          <x>56</x>
          <y>139</y>
          <width>133</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="labelName">
        <property name="geometry">
         <rect>
          <x>24</x>
          <y>16</y>
          <width>27</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Name</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineSurname">
        <property name="geometry">
         <rect>
          <x>257</x>
          <y>16</y>
          <width>131</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEmail">
        <property name="geometry">
         <rect>
          <x>57</x>
          <y>49</y>
          <width>331</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="labelAge">
        <property name="geometry">
         <rect>
          <x>229</x>
          <y>112</y>
          <width>19</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Age</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineAddress">
        <property name="geometry">
         <rect>
          <x>57</x>
          <y>79</y>
          <width>331</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="labelSurname">
        <property name="geometry">
         <rect>
          <x>199</x>
          <y>16</y>
          <width>42</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Surname</string>
        </property>
       </widget>
       <widget class="QLabel" name="labelEmail">
        <property name="geometry">
         <rect>
          <x>24</x>
          <y>49</y>
          <width>28</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>E-mail</string>
        </property>
       </widget>
       <widget class="QLabel" name="labelAddress">
        <property name="geometry">
         <rect>
          <x>13</x>
          <y>79</y>
          <width>39</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Address</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="linePhone">
        <property name="geometry">
         <rect>
          <x>56</x>
          <y>109</y>
          <width>131</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="labelPhone">
        <property name="geometry">
         <rect>
          <x>21</x>
          <y>109</y>
          <width>30</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Phone</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineName">
        <property name="geometry">
         <rect>
          <x>58</x>
          <y>20</y>
          <width>133</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="labelNickname">
        <property name="geometry">
         <rect>
          <x>5</x>
          <y>139</y>
          <width>45</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Nickname</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineAge">
        <property name="geometry">
         <rect>
          <x>256</x>
          <y>111</y>
          <width>131</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>116</x>
         <y>204</y>
         <width>239</width>
         <height>34</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Copyright Â© 2017  All rights reserved by Mustafa</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Teyfik AVKAN</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QLabel" name="labelUserImg2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>180</y>
         <width>101</width>
         <height>101</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>Edit</string>
      </attribute>
      <widget class="QPushButton" name="saveBttn">
       <property name="geometry">
        <rect>
         <x>117</x>
         <y>261</y>
         <width>81</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>SaveInfo</string>
       </property>
      </widget>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="geometry">
        <rect>
         <x>5</x>
         <y>4</y>
         <width>401</width>
         <height>171</height>
        </rect>
       </property>
       <property name="title">
        <string>Informations of Users</string>
       </property>
       <widget class="QLabel" name="labelName_3">
        <property name="geometry">
         <rect>
          <x>24</x>
          <y>16</y>
          <width>27</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Name</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEditSurname">
        <property name="geometry">
         <rect>
          <x>257</x>
          <y>16</y>
          <width>131</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEditEmail">
        <property name="geometry">
         <rect>
          <x>57</x>
          <y>49</y>
          <width>331</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="labelAge_3">
        <property name="geometry">
         <rect>
          <x>229</x>
          <y>110</y>
          <width>19</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Age</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEditPassword">
        <property name="geometry">
         <rect>
          <x>159</x>
          <y>139</y>
          <width>131</width>
          <height>20</height>
         </rect>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEditAddress">
        <property name="geometry">
         <rect>
          <x>57</x>
          <y>79</y>
          <width>331</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="labelSurname_3">
        <property name="geometry">
         <rect>
          <x>199</x>
          <y>16</y>
          <width>42</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Surname</string>
        </property>
       </widget>
       <widget class="QSpinBox" name="spinBoxEditAge">
        <property name="geometry">
         <rect>
          <x>257</x>
          <y>109</y>
          <width>131</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="labelEmail_3">
        <property name="geometry">
         <rect>
          <x>24</x>
          <y>49</y>
          <width>28</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>E-mail</string>
        </property>
       </widget>
       <widget class="QLabel" name="labelAddress_3">
        <property name="geometry">
         <rect>
          <x>13</x>
          <y>79</y>
          <width>39</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Address</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEditPhone">
        <property name="geometry">
         <rect>
          <x>56</x>
          <y>109</y>
          <width>131</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="labelPhone_3">
        <property name="geometry">
         <rect>
          <x>21</x>
          <y>109</y>
          <width>30</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Phone</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEditName">
        <property name="geometry">
         <rect>
          <x>58</x>
          <y>20</y>
          <width>133</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="labelPassword">
        <property name="geometry">
         <rect>
          <x>108</x>
          <y>139</y>
          <width>46</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Password</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="layoutWidget_3">
       <property name="geometry">
        <rect>
         <x>116</x>
         <y>204</y>
         <width>239</width>
         <height>34</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Copyright Â© 2017  All rights reserved by Mustafa</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Teyfik AVKAN</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QPushButton" name="savePassword">
       <property name="geometry">
        <rect>
         <x>249</x>
         <y>261</y>
         <width>81</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>SavePassword</string>
       </property>
      </widget>
      <widget class="QLabel" name="labelUserImg">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>180</y>
         <width>101</width>
         <height>101</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Notes</string>
    </attribute>
    <widget class="QTabWidget" name="tabWidget_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>441</width>
       <height>321</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tab_5">
      <attribute name="title">
       <string>My Notes</string>
      </attribute>
      <widget class="QTableView" name="tableView">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>311</width>
         <height>261</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushShowBttn">
       <property name="geometry">
        <rect>
         <x>330</x>
         <y>10</y>
         <width>91</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Show</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBoxDelete">
       <property name="geometry">
        <rect>
         <x>330</x>
         <y>212</y>
         <width>91</width>
         <height>23</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushDeleteBttn">
       <property name="geometry">
        <rect>
         <x>330</x>
         <y>246</y>
         <width>91</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBoxOpen">
       <property name="geometry">
        <rect>
         <x>330</x>
         <y>97</y>
         <width>91</width>
         <height>23</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushOpenBttn">
       <property name="geometry">
        <rect>
         <x>330</x>
         <y>127</y>
         <width>91</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Open</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_6">
      <attribute name="title">
       <string>Add Note</string>
      </attribute>
      <widget class="QPushButton" name="pushAddBttn">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>265</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
      <widget class="QPlainTextEdit" name="textNote">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>52</y>
         <width>411</width>
         <height>201</height>
        </rect>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>11</y>
         <width>411</width>
         <height>24</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="labelSubject">
            <property name="text">
             <string>Subject :</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineSubject"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="labelDate">
            <property name="text">
             <string>Date:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDateTimeEdit" name="dateTimeEdit"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_8">
      <attribute name="title">
       <string>Search</string>
      </attribute>
      <widget class="QPushButton" name="pushSearchBttn">
       <property name="geometry">
        <rect>
         <x>335</x>
         <y>41</y>
         <width>91</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Search</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEditSearch">
       <property name="geometry">
        <rect>
         <x>335</x>
         <y>11</y>
         <width>91</width>
         <height>23</height>
        </rect>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBoxSearchOpen">
       <property name="geometry">
        <rect>
         <x>335</x>
         <y>117</y>
         <width>91</width>
         <height>23</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushOpenSearchedBttn">
       <property name="geometry">
        <rect>
         <x>335</x>
         <y>151</y>
         <width>91</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Open</string>
       </property>
      </widget>
      <widget class="QTableWidget" name="tableWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>10</y>
         <width>321</width>
         <height>261</height>
        </rect>
       </property>
       <column>
        <property name="text">
         <string>uniquenote</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>subject</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>note</string>
        </property>
       </column>
      </widget>
      <widget class="QComboBox" name="comboBoxSearchDelete">
       <property name="geometry">
        <rect>
         <x>335</x>
         <y>214</y>
         <width>91</width>
         <height>23</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushDeleteSearchedBttn">
       <property name="geometry">
        <rect>
         <x>335</x>
         <y>249</y>
         <width>91</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_7">
    <attribute name="title">
     <string>Reminder</string>
    </attribute>
    <widget class="QTableView" name="tableViewDate">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>311</width>
       <height>291</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushShowDatesBttn">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>130</y>
       <width>91</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Show Dates</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
